Sources/Lumina/Camera/Extensions/SessionConfigurationExtension.swift

```diff
@@ -0,0 +1,230 @@
+//
+//  SessionConfigurationExtension.swift
+//  Lumina
+//
+//  Created by David Okun on 11/20/17.
+//  Copyright © 2017 David Okun. All rights reserved.
+//
+
+import Foundation
+import AVFoundation
+
+extension LuminaCamera {
+  func requestVideoPermissions() {
+    self.sessionQueue.suspend()
+    AVCaptureDevice.requestAccess(for: .video) { success in
+      if success {
+        LuminaLogger.notice(message: "successfully enabled video permissions")
+        self.sessionQueue.resume()
+        self.delegate?.cameraSetupCompleted(camera: self, result: .requiresUpdate)
+      } else {
+        LuminaLogger.warning(message: "video permissions were not allowed - video feed will not show")
+        self.delegate?.cameraSetupCompleted(camera: self, result: .videoPermissionDenied)
+      }
+    }
+  }
+  
+  func requestAudioPermissions() {
+    self.sessionQueue.suspend()
+    AVCaptureDevice.requestAccess(for: AVMediaType.audio) { success in
+      if success {
+        LuminaLogger.notice(message: "successfully enabled audio permissions")
+        self.sessionQueue.resume()
+        self.delegate?.cameraSetupCompleted(camera: self, result: .requiresUpdate)
+      } else {
+        LuminaLogger.warning(message: "audio permissions were not allowed - audio feed will not be present")
+        self.delegate?.cameraSetupCompleted(camera: self, result: .audioPermissionDenied)
+      }
+    }
+  }
+  
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/dokun1/Lumina/blob/5e323e0c2296f8f5c5b041c942e35a2720d9c81e/Sources/Lumina/Camera/Extensions/SessionConfigurationExtension.swift#L40"></span>Lines should not have trailing whitespace.<br />
<code>trailing_whitespace</code> <code>SessionConfigurationExtension.swift:40</code></td>
    </tr>
  </tbody>
</table>